client/components/wizard-bar/progress-icon.jsx

```diff
@@ -0,0 +1,10 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+export default () => (
```

Comment:
Could you do something like:
```
const ProgressIcon = () => { }
export default ProgressIcon;
```
instead? That way `displayName` is set by babel and it's easier to debug and inpect components. I doubt this one will cause anyone any troubles, but it's just good practice - otherwise it shows up as `Unknown` in React DevTools:
<img width="627" alt="screen shot 2017-05-25 at 20 20 55" src="https://cloud.githubusercontent.com/assets/3392497/26464327/aed764d6-4187-11e7-859b-eed723e783ed.png">